packages/web-components/src/components/va-date/va-date.css

```diff
@@ -2,6 +2,21 @@
 @import '../../mixins/form-field-error.css';
 @import '../../mixins/hint-text.css';
 
+.usa-label--required {
+  color: var(--v3-color-error-dark);
+}
+
+.usa-error-message {
+  color: var(--v3-color-error-dark);
+}
+
+:host([uswds][error]:not([error=''])) va-select::part(select),
+:host([uswds][error]:not([error=''])) va-text-input::part(input) {
+  border: 4px solid var(--v3-color-error-dark);
+}
```

Comment:
Is `va-date` used in `va-memorable-date`? I'm not sure why this component is being updated.

I'm also not sure why we would need to add error styling in a parent component for the children. Are we not able to activate the error state in `va-select` and `va-text-input` (the child components used in `va-memorable-date`) so that it naturally displays the error border for their v3 variation?

Reply:
And btw the answer to that question about styling the child components could be yes (like we had to do it that way). I just want to double check that we tried doing it with the child component directly as well.